app/assets/javascripts/countries/views/CountryShowView.js

```diff
@@ -472,7 +472,14 @@ define([
       var $el = $('.country-burned_forests');
       var $graph = $('.burned_forests-graph');
       var $comingSoon = $el.find('.coming-soon');
-      var json = $graph.data('json');
+      var json = _.filter(_.map($graph.data('json'), function(el) {
+        if (el.year == 2003 || el.year == 2004 || el.year == 2011 || el.year == 2012) {
```

Comment:
Ahh gotcha, I saw that they were objects so I didn't worry about the potential for 0s or other false values! Good point.